### DISCUSSION TITLE
Does it support multimodal model with cnt , cat, txt, img features?

### DISCUSSION THREAD

**Discussion by wjlgatech** [oegedijk/explainerdashboard#180] [Q&A] [answered]
Does this package support multimodal model with cnt , cat, txt, img features?

Also I have Tensorflow based models, do you have some automation to wrap it into sklearn compatible model?

Thank you for this amazing package!

**Comment by oegedijk** [comment, marked as answer]
Hi @wjlgatech,

the dashboard is intended for standard tabular usecases: so definitely a mix of continuous and categorical features are supported, but not image nor text really (although I guess you could throw a big tabular classifier at tf-idf a dataset and run it through the dashboard)

**Comment by oegedijk** [comment]
tensorflow is also currently not supported, but pytorch models are supported through the `skorch` package wrappers...
